Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -178,6 +196,13 @@
 	end
 end
 
+% For GE and multi-PLD or single-PLD not defined in the GELabelingDurationField, we prefer LabelingDuration from study par due to issues with eASL
+if strcmp(jsonIn.Manufacturer, 'GE') && isfield(studyPar,'LabelingDuration') && ~isequal(studyPar.LabelingDuration,jsonOut.LabelingDuration) &&...
+	( length(jsonOut.LabelingDuration)>1 || ~isfield(jsonIn,'GELabelingDuration'))
+	warning('Labeling duration differs in DICOM and studyPar.');
```

Comment:
differs between. Otherwise, it could strictly speaking differ within the DICOM (having two values) and within the studyPar (having two values)

Reply:
Fixed in 43d03f7b